Tasks/BelkevichAleksandr/Task2/Index.py

```diff
@@ -0,0 +1,135 @@
+import datetime
+
+def time_selection():
+    '''
+    function for checking selection input from user
+    '''
+    while True:
+        try:
+            a = int(input("Please enter 1 for now time or 2 for enter your time: "))
+
+            # check if the entered number is correct
+            if a == 1 or a == 2:
+                return a
+
+            # output for explanation
+            print(f"Error, {a} isn't 1 or 2")
+        except:
+            print("Error, not an integer entered")
+            continue
+
+def input_time_user():
+    '''
+    function for checking input time from user
+    '''
+    while True:
+        try:
+            a = input("Please enter time in this format (HH:MM): ")
+            s = corrected_time(a)
+
+            # check if the entered time is correct
+            if 0 <= s[0] * 10 + s[1] < 24 and 0 <= s[2] * 10 + s[3] < 60 and len(s) == 4:
+                return s
```

Comment:
I would start with some validation first, simple is better then complex, and we can give a user some piece of advice on how to get a proper result, like 'you've missed : ' or 'the numbers are too low/high' and so on